File: app/src/main/java/com/grace/placessearch/data/model/VenueResponse.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class VenueResponse {

    @SerializedName("meta")
    @Expose
    private Meta meta;
    @SerializedName("response")
    @Expose
    private SingleVenueResponse singleVenueResponse;

    public Meta getMeta() {
        return meta;
    }

    public void setMeta(Meta meta) {
        this.meta = meta;
    }

    public SingleVenueResponse getSingleVenueResponse() {
        return singleVenueResponse;
    }

    public void setSingleVenueResponse(SingleVenueResponse singleVenueResponse) {
        this.singleVenueResponse = singleVenueResponse;
    }
}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/VenueResponse.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class VenueResponse {

    @SerializedName("meta")
    @Expose
    var meta: Meta? = null
    @SerializedName("response")
    @Expose
    var singleVenueResponse: SingleVenueResponse? = null
}